docs/about/releases.mdx

```diff
@@ -3,18 +3,18 @@ title: "Releases"
 description: "Latest new features and changelog."
 ---
 
-<Card
+<Card>
```

Comment:
@thomaschung408 The end bracket `>` here shouldn't be changed or it may mess up the formatting of this: 
![image](https://user-images.githubusercontent.com/78053898/225103951-04916423-902b-454f-aaa8-9f1fe95253e6.png)

Reply:
You can look [here](https://github.com/mage-ai/mage-ai/pull/2195/files) to see what it was previously.